lib/bolt/transport/ssh.rb

```diff
@@ -9,7 +9,7 @@ module Bolt
   module Transport
     class SSH < Base
       def self.options
-        %w[port user password sudo-password private-key host-key-check connect-timeout tmpdir run-as]
+        %w[port user password sudo-password private-key host-key-check connect-timeout tmpdir run-as run-as-command]
```

Comment:
We should add a section in `validate` that asserts `run-as-command` is an array of strings.